docs/source/overview/quicktour.md

```diff
@@ -94,144 +94,63 @@ Now, let's take a look at the json configuration file you need to provide to opt
+                    "sub_types": [{"name": "avg"}]
+                }
+            ]
+        }
+    },
+    "passes": {
+        "onnx_conversion": {
+            "type": "OnnxConversion",
             "config": {
-                "model_path": "path",
-                "inference_settings": null
+                "target_opset": 13
             }
         },
-        "from_pass": "OrtTransformersOptimization",
-        "pass_run_config": {
-            "model_type": "bert",
-            "num_heads": 0,
-            "hidden_size": 0,
-            "optimization_options": null,
-            "opt_level": null,
-            "use_gpu": false,
-            "only_onnxruntime": false,
-            "float16": false,
-            "input_int32": false,
-            "use_external_data_format": false
-        },
-        "is_pareto_frontier": true,
-        "metrics": {
-            "value": {
-                "accuracy": 0.8602941036224365,
-                "latency": 87.4454
-            },
-            "cmp_direction": {
-                "accuracy": 1,
-                "latency": -1
-            },
-            "is_goals_met": true
+        "quantization": {
+            "type": "OnnxDynamicQuantization"
         },
-        "date_time": 1681211541.682187
-    }
-}
-```
-
-You can also call the following methods to plot the Pareto frontier footprints. Also please make sure you installed `plotly` successfully.
-```python
-from olive.engine.footprint import Footprint
-footprint = Footprint.from_file("footprints.json")
-footprint.plot_pareto_frontier()
-```
-### Olive Packaging
-Olive also can package output artifacts when user adds `PackagingConfig` to Engine configurations.
-```
-"engine": {
-    ...
-    "packaging_config": {
-        "type": "Zipfile",
-        "name": "OutputModels"
     },
-    ...
-}
-```
-Olive packaging will generate a ZIP file which includes 3 folders: `CandidateModels`, `SampleCode` and `ONNXRuntimePackages`:
-* `CandidateModels`: top ranked output model set
-    * Model file
-    * Olive Pass run history configurations for candidate model
-    * Inference settings (`onnx` model only)
-* `SampleCode`: code sample for ONNX model
-    * C++
-    * C#
-    * Python
-* `ONNXRuntimePackages`: ONNXRuntime package files with the same version that were used by Olive Engine in this workflow run.
-
-Please refer to [Packaging Olive artifacts](../tutorials/packaging_output_models.md) for more details.
-
-## Azure ML Client
-
-If you will use Azure ML resources and assets, you need to provide your Azure ML client configurations. For example:
-* You have AzureML system for targets or hosts.
-* You have Azure ML model as input model.
-
-AzureML authentication credentials is needed. Refer to
-[this](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-setup-authentication?tabs=sdk)  for
-more details.
-
-You can either add configurations to the Olive json config file:
-```json
-"azureml_client": {
```

Comment:
Do we want to move the example config to https://microsoft.github.io/Olive/overview/options.html#azure-ml-client? Otherwise there is no example config in the doc for AML configuration.

Reply:
We should have separate how to for that. Need to include it quick tour which should be as brief as possible.

Reply:
Yeah that's what I mean. I mean we could move this example configuration part from `quicktour` to `options` `azure-ml-client` section. Otherwise the document will not include AML configuration examples.